app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -191,13 +195,28 @@ export const SendBitcoinScreen: ScreenType = ({
       memo: memoInvoice,
       paymentType,
       address,
+      lnurl,
       sameNode,
     } = validPayment(destination, tokenNetwork, client)
 
     if (valid) {
       setAddress(address)
       setPaymentType(paymentType)
       setInvoice(invoice)
+
+      if (lnurl) {
+        setPaymentType("lnurl")
+        const lnurlParams = {
+          lnurl: lnurl, 
+          minSendable: (route.params?.lnurlParams.minSendable/1000)+"", 
+          maxSendable: (route.params?.lnurlParams.maxSendable/1000)+"", 
```

Comment:
Numbers should be of type `number` in TS. Please do not cast numbers to strings.